Documentation/HowToGuides/CreatingProxyActionsAndChainingActions/README.md

```diff
@@ -0,0 +1,82 @@
+
+The Zinnia Actions can be used as proxies to enable chaining of Actions to all drive the same outcome defined in one place.
+
+Following on from the scene created in Adding A Unity Button Action, we’re going to move the Action functionality to a Proxy Action and then chain the button press Action and keyboard Action to this newly created Proxy Action. This will move our output functionality into one place and allow us to call it from many different actions.
+
+### Prerequistes 
+
+* A Unity Button Action exists in the scene. See [Using The Unity Button Action].
+
+## Let's Start
+
+### Step 1
+
+Create an empty `GameObject` in the Unity Hierarchy and rename it to `FloorToggler` then click the `Add Component` button and select the `Boolean Action` component.
+This newly created `Boolean Action` will be the action that defines how we toggle the active state of the `Floor` GameObject.
+
+![Creating The Floor Toggler](assets/images/CreatingTheFloorToggler.png)
+
+### Step 2
+
+Click the `+` symbol in the bottom right corner of the `Activated` event parameter in the `Boolean Action` component and then drag and drop the `Floor` GameObject into the box that appears and displays `None` `(Object)`.
+
+![Creating The Boolean](assets/images/CreatingTheBoolean.png)
+
+### Step 3
+
+Do the same as Step 2 but for the `Deactivated` event parameter in the `Boolean Action` component, but tick the checkbox under the `GameObject.SetActive function`.
+
+![Deactivating The Boolean](assets/images/DeactivatingTheBoolean.png)
+
+### Step 4
+
+Now we have a generic Action that will disable the `Floor` GameObject when the Action receives `true` and will re-enable the `Floor` GameObject when the Action receives `false`.
+
+We could hook up the `Unity Button Action` `Value Changed` event to simply call the `Receive` method on our `FloorToggler` GameObject `BooleanAction` component and this will pass the state of the `Unity Button Action` over to the `Boolean Action` whenever the value changes. However, doing this means for every input action, we’d need to wire up their `Value Changed` event to point to the `Receive` method on our `BooleanAction` component. This can again get very cumbersome and hard to maintain, so instead we’re going to use the `Sources` parameter that is available on a `Zinnia.Action.`
+
+The `Sources` parameter allows us to specify other Actions for our `BooleanAction` to monitor changes on and re-emit any events that happen on those source actions.
+
+Expand the `Sources parameter` and enter the desired number of other Actions to monitor in the `Size` parameter field. In this example, we want to watch two other Actions (the Menu Button `Unity Button Actio` and the Space Key `Unity Button Action`) so enter `2` into the Sizes parameter field.
```

Comment:
`Sources parameter` should be `Sources` parameter (parameter not in backticks)

Also look at this line:


> we want to watch two other Actions (the Menu Button `Unity Button Actio` and the Space Key `Unity Button Action`)

It's actually now the space key and the xbox controller button A action so this needs to be worded accordingly.

Reply:
Space key is now action zero (so any reference to Menu button should now refer to the Space key instead)

Xbox button A is now action one (so any reference to Space key should now refer to the Xbox button A instead)